using-mobile-extensions/adobe-analytics-mobile-services/README.md

```diff
@@ -291,6 +363,28 @@ Use the following API to track a push messaging click through in Adobe Analytics
 }
 ```
 {% endtab %}
+
+{% tab title="iOS — Swift" %}
+Use the following API to track a push messaging click through in Adobe Analytics.
```

Comment:
Take out "through", so it is "click in Adobe Analytics".

Reply:
fixed